addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -211,11 +211,12 @@
                                                 <span class="s_website_form_label_content">Your Email</span>
                                                 <span class="s_website_form_mark"> *</span>
                                             </label>
-                                            <div class="col-sm">
+                                            <div class="col-sm">      
```

Comment:
Here we have a useless diff, so we try to avoid those as much as possible to make reviews a bit easier, so that we know that modified lines are relevant.